.dockerignore

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+*.iml
+.eggs
+.github
+.git
+build
+apache_airflow.egg-info
```

Comment:
Can we add:

```
dist/
docs/
```

Reply:
I added dist, but I think we should not skip docs - otherwise no-one will be able to build docs using this image (which I think is useful). What I did instead - i added docs/_build dir - to exclude the generated doc files - which is 78 MB out of the 87 MB for the whole doc folder (after the docs are generated)

Reply:
That's probably what I meant. Sounds good.